src/lua/lua.cpp

```diff
@@ -264,12 +270,6 @@ int LuaAPI::get_province(lua_State * L) {
 	return 4;
 }
 
-#ifndef SERVER_HEADLESS
```

Comment:
why are you removing this, this is needed for updating provinces

Reply:
I have modef it up there with the other headers, as  I needed to check if all headers are win or linux dependent, was easier to check.